test/runtests.jl

```diff
@@ -8,13 +8,14 @@ end
 
 @testset "Sealed boxes" begin
     msg = "Message"
-    msg_len = length(msg)
+    msg_len = sizeof(msg)
     ciphertext_len = crypto_box_SEALBYTES + msg_len
     recipient_pk = Vector{Cuchar}(undef, crypto_box_PUBLICKEYBYTES)
     recipient_sk = Vector{Cuchar}(undef, crypto_box_SECRETKEYBYTES)
     @test crypto_box_keypair(recipient_pk, recipient_sk) == 0
     ciphertext = Vector{Cuchar}(undef, ciphertext_len)
     @test crypto_box_seal(ciphertext, msg, msg_len, recipient_pk) == 0
+    @test crypto_box_seal(msg, recipient_pk) == ciphertext
```

Comment:
I don't know why this test case won't pass. Should these two `ciphertext`s always be the same?